src/ios/BranchSDK.m

```diff
@@ -122,9 +122,7 @@ - (void)disableTracking:(CDVInvokedUrlCommand*)command
 {
 
   bool enabled = [[command.arguments objectAtIndex:0] boolValue] == YES;
```

Comment:
This should be:
`bool enabled = [[command.arguments objectAtIndex:0] boolValue];`